conf/messages.en

```diff
@@ -1621,7 +1663,7 @@ accessibilityStatement.reporting.pre.link = We are always looking to improve the
 accessibilityStatement.reporting.link = accessibility problem (opens in a new window or tab)
 
 accessibilityStatement.what.to.do = What to do if you are not happy with how we respond to your complaint
-accessibilityStatement.what.to.do.pre.eass.link = The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the 'accessibility regulations'). If you are not happy with how we respond to your complaint,
+accessibilityStatement.what.to.do.pre.eass.link = The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ’accessibility regulations’). If you are not happy with how we respond to your complaint,
```

Comment:
These are the wrong quotes for this type of text? It's not "possessive", the regulations is now "owned".
Make this:
```
Accessibility Regulations 2018 (the ''accessibility regulations'')
```
The double `''` ensures one escapes and renders the second.